data/json/items/comestibles/med.json

```diff
@@ -1486,8 +1486,8 @@
     "id": "thorazine",
     "type": "COMESTIBLE",
     "comestible_type": "MED",
-    "name": { "str_sp": "Thorazine" },
-    "description": "Anti-psychotic medication.  Used to stabilize brain chemistry, it can arrest hallucinations and other symptoms of psychosis.  Carries a sedative effect.  Its generic name is chlorpromazine.",
+    "name": { "str_sp": "chlorpromazine" },
```

Comment:
Kaluptic Psychosis needs its description changed to this.

Reply:
come again?

Reply:
The trait references Thorazine in it's description, it now needs to reference chlorpromazine.

Reply:
aaah found it. Changing it!